mrtt-ui/src/styles/containers.js

```diff
@@ -35,4 +36,23 @@ const LinkCard = styled(Link)(({ theme: themeMui }) => ({
   color: theme.color.text
 }))
 
-export { PagePadding, RowSpaceBetween, LinkCard, RowCenterCenter, RowFlexEnd, ButtonContainer }
+const PaddedPageTopSection = styled(Stack)(({ theme: themeMui }) => ({
+  padding: themeMui.spacing(2),
+  borderBottom: theme.border.primary,
+  backgroundColor: theme.color.bodyBackground
+}))
```

Comment:
I dont like object styles, but using a template string for styles when utilizing a theme prop doesnt format nicely.